.netlify/build/chunks/Tags_BqEAc-1g.mjs

```
@@ -0,0 +1,188 @@
+  const enToRu = {
+    a: "а",
+    b: "б",
+    v: "в",
+    g: "г",
+    d: "д",
+    e: "е",
+    yo: "ё",
+    zh: "ж",
+    z: "з",
+    i: "и",
+    y: "й",
```

Biome (1.9.4)

[error] This property value named e is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named e


(lint/suspicious/noDuplicateObjectKeys)